bin/jack.js

```diff
@@ -692,6 +691,7 @@ Much more documentation available at: https://www.node-tap.org/
                   appended to the filenames.`,
   }),
 
+  /* eslint-disable-next-line */
```

Comment:
I'm not sure what to do here. There's a lot of code in here and it's hard to see it all. So I just disable the warning. However, if it is accurate, then I think this key will take precedence over the first definition.

Reply:
To be clear: this comment directive is disabling a "duplicate key" warning.